code/workspaces/infrastructure-api/resources/jupyter.deployment.template.yml

```diff
@@ -77,9 +79,14 @@ spec:
             periodSeconds: 10
 {{#volumeMount}}
           volumeMounts:
+            - name: spark-kubernetes-properties
+              mountPath: '$SPARK_HOME/conf/spark-defaults.conf'
```

Comment:
Does the use of `$SPARK_HOME` here work? I don't think that Kubernetes manifests can use env variables like this so we're probably going to need another replacement parameter

Reply:
Yes that's right. I would replace it with the full path